jwst/jump/tests/test_miri_first_lastframe.py

```diff
@@ -0,0 +1,82 @@
+"""
+Unit test for jump step to test not using MIRI first frame or last frame if set to DO_NOT_USE
+"""
+import pytest
+import numpy as np
+from jwst.jump import JumpStep
+from jwst import datamodels
+from jwst.datamodels import dqflags, MIRIRampModel
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+
+
+@pytest.fixture(scope='function')
+def science_image():
+    """Generate science image"""
+    # size of integration
+    nints = 1
+    ngroups = 7
+    xsize = 1032
+    ysize = 1024
+
+    # create the data and groupdq arrays
+    # create a JWST datamodel for MIRI data
+    im = MIRIRampModel((nints, ngroups, ysize, xsize))
+    im.meta.instrument.detector = 'MIRIMAGE'
+    im.meta.instrument.name = 'MIRI'
+    im.meta.subarray.name = 'FULL'
+    im.meta.subarray.xstart = 1
+    im.meta.subarray.ystart = 1
+    im.meta.subarray.xsize = 1032
```

Comment:
Again, reduce size. Don't need a full frame to test a million pixels with identical values.